code/client/src/components/LocalImport.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import { ImportOutlined, LoadingOutlined } from '@ant-design/icons'
+import { Button, Upload } from 'antd'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router'
+import message from '../message'
+import storage from '../storage'
+import util from '../util'
+import ONEUtil from '../../../lib/util'
+import { walletActions } from '../state/modules/wallet'
+import Paths from '../constants/paths'
+import {LocalExportMessage} from './LocalExport'
+
+const LocalImport = () => {
+  const dispatch = useDispatch()
+  const history = useHistory()
+  const wallets = useSelector(state => state.wallet.wallets)
+  const [fileUploading, setFileUploading] = useState(false)
+
+  const getDataFromFile = file =>
+    new Promise(resolve => {
+      const reader = new FileReader()
+      reader.addEventListener('load', () => resolve(reader.result))
+      reader.readAsArrayBuffer(file)
```

Comment:
Since this does not throw any error, we should have an error listener as well to reject the promise